setup.cfg

```diff
@@ -18,3 +18,7 @@ warn_unused_ignores = true
 
 [mypy-tests.*]
 disallow_untyped_defs = false
+
+[options]
+package_dir =
+    sentry_pypi = .
```

Comment:
this makes this an installable package -- which I intentionally did not want because it's not a library it's a collection of scripts

Reply:
devenv doesn't work otherwise, and that was your instructions.
Please revise instructions, or revise your not-installable requirement.

Reply:
it works, just produces an error message -- arguably a bug in `tox`

Reply:
My vote is to allow the "collection of scripts" to be installed.
`python -m add_pkg` is only incidentally working if you insist that these aren't libraries -- `-m` is explicitly the option to run libraryies as scripts.
I'm -1 for "it works, just with an error".

Reply:
`-m` is to run *modules* as scripts, it has nothing to do with whether the python files are packagable and distributable with setuptools

Reply:
tomato tomato
We disagree.